.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: ruby
 script: bundle exec rake
 rvm:
-  - ruby-head
```

Comment:
sometimes I comment it out and explain why it's not there:

#   - ruby-head # unf_ext gem won't build under ruby-head